### PR TITLE
Fill currencies add as subunits format

### PR DESCRIPTION
**Pull request by danielwitz**
fix for this [issue](https://github.com/Rhymond/go-money/issues/49)

### CODE REVIEW

File: formatter.go [hunk 1]
```diff
@@ -54,6 +55,11 @@ func (f *Formatter) Format(amount int64) string {
 	return sa
 }
 
+// ToMajorUnits returns float64 representing the value in sub units using the currency data
+func (f *Formatter) ToMajorUnits(amount int64) float64 {
```
Comment (Rhymond) [inline formatter.go]: Hi,

It's really project specific feature. You can do it by extending money struct in your project.
Or just get amount by calling `Amount()` and do calculations later

File: go.mod [hunk 2]
```diff
@@ -1 +1 @@
-module github.com/Rhymond/go-money
+module github.com/danielwitz/go-money
```
Comment (Rhymond) [inline go.mod]: I don't think this one should be here

File: currency.go [hunk 3]
```diff
@@ -4,7 +4,7 @@ import (
 	"strings"
 )
 
-// Currency represents money currency information required for formatting.
+// Currency represents money currency information required for formatting
```
Comment (Rhymond) [inline currency.go]: See golang.org/doc/effective_go.html#commentary. Comments documenting declarations should be full sentences, even if that seems a little redundant. This approach makes them format well when extracted into godoc documentation. Comments should begin with the name of the thing being described and end in a period

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Comment (danielwitz) [reply currency.go]: yeah, not sure why I deleted them.. added them back

File: formatter.go (hunk 1 above)
Comment (danielwitz) [reply formatter.go]: I disagree, a lot of other money libs in other languages have that [feature](https://github.com/RubyMoney/money/blob/master/lib/money/money.rb#L307) as its a fairly common use case.

File: go.mod (hunk 2 above)
Comment (danielwitz) [reply go.mod]: Yeah, it so I can use my fork locally. 
You should omit it if you merge it.

**Comment by danielwitz** [comment]
@Rhymond Thanks for looking at my PR, you can check the [Money gem's currency data](https://github.com/RubyMoney/money/blob/master/config/currency_iso.json) and see if I got something wrong.

**Pull request closed without merging**